packages/mcp-workflow/src/nodes/getUserInput/factory.ts

```diff
@@ -70,7 +70,7 @@ export function createGetUserInputNode<TState extends StateType<StateDefinition>
         reason: `Property '${propertyName}' is missing from the workflow state.`,
       };
     },
-    userInputProperty,
+    userInputProperty = 'userInput' as keyof TState,
```

Comment:
`'userInput'` was supposed to be the default value. It got refactored out at some point.

Reply:
We should. It's [already implicit](https://github.com/forcedotcom/mobile-mcp-tools/blob/679bd5d2bc2f37ed1dcd7ce2195ff3d94219a056/packages/mcp-workflow/src/tools/base/abstractWorkflowTool.ts#L104) in code like `AbstractWorkflowTool` and the [orchestrator metadata](https://github.com/forcedotcom/mobile-mcp-tools/blob/679bd5d2bc2f37ed1dcd7ce2195ff3d94219a056/packages/mcp-workflow/src/tools/orchestrator/metadata.ts#L23), so it's effectively required to be a part of the workflow state.

I'll look at what it takes to create a derived type that can require that property.